### PR TITLE
[zh-cn] update update-api-object-kubectl-patch.md

### PR DESCRIPTION
**Pull request by Zhuzhenghao**
Sync the file with [content/en/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch.md](https://github.com/kubernetes/website/blob/main/content/en/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch.md)

Fix formats

### CODE REVIEW

File: content/zh-cn/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ in this task demonstrate a strategic merge patch and a JSON merge patch.
 Here's the configuration file for a Deployment that has two replicas. Each replica
 is a Pod that has one container:
 -->
-## 使用策略合并 patch 更新 Deployment    {#use-a-strategic-merge-patch-to-update-a-deployment}
+## 使用策略合并 patch 更新 Deployment {#use-a-strategic-merge-patch-to-update-a-deployment}
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch.md:36]: this kind of changes are unnecessary.

The extra spaces (4-6) are for readability's sake.

File: content/zh-cn/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch.md [hunk 2]
```diff
@@ -631,7 +631,7 @@ kubectl patch deployment patch-demo --patch '{"spec": {"template": {"spec": {"co
 <!--
 ### Update an object's replica count using `kubectl patch` with `--subresource` {#scale-kubectl-patch}
 -->
-### 使用 `kubectl patch` 和 `--subresource` 更新一个对象的副本数   {#scale-kubectl-patch}
+### 使用 `kubectl patch` 和 `--subresource` 更新一个对象的副本数 {#scale-kubectl-patch}
 
 {{< feature-state for_k8s_version="v1.24" state="alpha" >}}
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch.md:637]: line 642 has upstream change to be sync'ed.

File: content/zh-cn/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch.md (hunk 1 above)
Comment (Zhuzhenghao) [reply content/zh-cn/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch.md]: Thanks for you time , Ihave remove these changes.

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**